### PR TITLE
Updated ACCESS_TOKEN_URL to api.instagram

### PR DESCRIPTION
**Pull request by matthowland**
https://instagram.com will result in a 405 error after last update of instagram api (nov 17)

### CODE REVIEW

**Comment by dvdhpkns** [comment]
I was just coming here to open up this issue -- it was driving me crazy. Thanks @matthowland. @omab can you please merge this and cut a new version?

File: social/backends/instagram.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 class InstagramOAuth2(BaseOAuth2):
     name = 'instagram'
     AUTHORIZATION_URL = 'https://instagram.com/oauth/authorize'
```
Comment (dvdhpkns) [inline social/backends/instagram.py]: @matthowland the AUTH URL needs to change as well.

**Comment by riklaunim** [comment]
Both url changed and it works.

**Comment by jgoclawski** [comment]
It affects us too, Instagram authentication has been broken for the past 4 days until we found this pull request. It works and fixes the problem.

BTW: we've changed both urls (AUTHORIZATION_URL, ACCESS_TOKEN_URL)

**Comment by omab** [comment]
@matthowland could you update ``AUTHORIZATION_URL`` too?

**Comment by jgoclawski** [comment]
There seems to be a duplicate of this PR, but with both URLs changed: https://github.com/omab/python-social-auth/pull/797.

**Comment by omab** [comment]
Fixed by #797

**Pull request closed without merging**